pages/education/school-resources.md

```diff
@@ -138,23 +138,118 @@ Resources for schools is a one-stop shop for School Certifying Officials (SCOs)
 
 [See past updates](https://www.benefits.va.gov/gibill/news.asp)
 </div>
-
-***
-
+<section class="usa-grid">
+    <div class="va-h-ruled--stars"></div>
+</section>
 <div>
   <h2 id="training-and-guides">Training and guides</h2>
 
-Get training and boost your skills using these resources fro School Certifying Officials (SCOs) serving military connected students.
-
-[Required 2019 training for School Certifying Officials (SCOs) >](https://www.benefits.va.gov/gibill/resources/education_resources/school_certifying_officials/online_sco_training.asp)
-
-Required if your school has 20 or more GI Bill students
-
+Get training and boost your skills using these resources for School Certifying Officials (SCOs) serving military-connected students.
+
+<b>[Required 2019 training for School Certifying Officials (SCOs) >](https://www.benefits.va.gov/gibill/resources/education_resources/school_certifying_officials/online_sco_training.asp)</b><br>
+Required if your school has 20 or more GI Bill students<br>
+    <div class="usa-accordion" data-multiselectable="false">
+        <ul class="usa-unstyled-list">
+            <li>
+                <button class="usa-accordion-button usa-button-unstyled" aria-controls="t_1">
+                    Essentials for School Certifying Officials
+                </button>
+                <div id="t_1" class="usa-accordion-content">
+                    <b>[School Official’s Handbook >](https://www.benefits.va.gov/GIBILL/docs/job_aids/SCO_Handbook.pdf)</b>     
+                    <br><b>[VA-ONCE quick reference guide >](https://www.benefits.va.gov/GIBILL/docs/vaonce/VAONCEguide.pdf)</b>  
+                    <br><b>[Find your Education Liaison Representative (ELR) >](https://www.benefits.va.gov/gibill/resources/education_resources/school_certifying_officials/elr.asp)</b>   
+                    <br><b>[Training webinars >](https://www.benefits.va.gov/gibill/resources/education_resources/school_certifying_officials/presentations.asp)</b>     
+                    <br><b>[WEAMS Public >](https://inquiry.vba.va.gov/weamspub/buildSearchInstitutionCriteria.do)</b>     
+                    <br><b>[Benefits A to Z >](https://benefits.va.gov/atoz/)</b>   
+                    <br><b>[Check your status as a Covered Institution >](https://www.benefits.va.gov/GIBILL/resources/education_resources/school_certifying_officials/Covered_Institutions.asp)</b>
+                    <br><br>
+                    <b>Reporting:</b>
+                    <ul>
```

Comment:
Can you just use a nested list here? Also, as Kenny pointed out, the normal markdown link here is underlined, but the prototype has non-underlined links, so may need to use html with css.

Reply:
change has been pushed.